coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild

```diff
@@ -24,6 +24,7 @@ DEPEND="
 	app-shells/bash-completion
 	coreos-base/hard-host-depends
 	coreos-base/coreos-sb-keys
+	coreos-devel/fero-client
```

Comment:
Maybe should in `sdk-extras` or jenkins will build it into the sdk every release. Makes more sense if just the people signing build it locally imo.